src/Markdig.Renderers.Docx/DocxDocumentRenderer.cs

```diff
@@ -90,7 +90,7 @@ public void ForceCloseParagraph()
         }
     }
 
-    public override DocxDocumentRenderer Render(MarkdownObject markdownObject)
+    public override object Render(MarkdownObject markdownObject)

```

Comment:
Changed the return type to object for compatibility with .NET Framework and .NET Standard 2.0:
Error CS8830  '...runtime doesn't support covariant return types in overrides. Return type must be 'object' to match overridden member 'RendererBase.Render(MarkdownObject)'